src/templates/style.css

```diff
@@ -245,3 +245,32 @@ i.fa-angle-right, i.fa-angle-down {
   border: 0;
   margin-top: 1em;
 }
+
+/* Dark mode support */
+body.dark {
+  background-color: #272525;
+}
+
+.dark .main {
+  color: white;
+}
+
+.dark .feature-button, .dark .scenario-button {
+  color: #ccc;
+  background-color: #23272A;
+}
+
+.dark .feature-button.active, .dark .feature-button:hover,
```

Comment:
Having these styled the same was making the buttons all run together (at least on my bad monitors).  What do you thing about something like:
```.css
.dark .feature-button.active, .dark .feature-button:hover {
  background-color: #272525;
  border-bottom-color: #23272A;
  color: #fff;
  font-weight: normal;
}
```

Reply:
If we do this then I think we can remove the `body.dark .feature-button.active, body.dark .feature-button:hover` block below.

Reply:
updated!